README.md

```diff
@@ -20,10 +20,66 @@ If you are looking to use ssb-server to run a pub, consider using [ssb-minimal-p
 
 ## Install
 
-To add `ssb-server` to your available CLI commands, install it using the `-g` global flag:
+How to Install `ssb-server` and create a working pub 
+
+1. `curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.34.0/install.sh | bash`
+
+2. `npm install -g node-gyp`
+
+3. `apt-get install autotools-dev automake`
+
+4. `nvm install 10`
+
+5. `nvm alias default 10`
+
+6. Then to add `ssb-server` to your available CLI commands, install it using the `-g` global flag:
 ```
 npm install -g ssb-server
 ```
+7. `nano ~/run-server.sh` and input:
+
+```
+#!/bin/bash
+while true; do
+  ssb-server start
+  sleep 3
+done
```

Comment:
be nice to have an explainer of what the purpose/function of this script is :)

Reply:
I added: 

> Be sure to start the pub server from this script, as this script will run the pub server and restart it even if it crashes.

What do you think?

Reply:
"Be sure to start the pub server from this script"
how about clarifying that that is step 10? :)